DALK.PL_ANALYZER/Models/Matches/MatchesModelView.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace DALK.PL_ANALYZER.Models.Matches
+{
+    public class MatchesModelView
+    {
+        public List<MatchModelView> Matches { get; private set; }
```

Comment:
Private set :)
Still if someone will take this collection can call .Clear() or .Add() - just saying, maybe its good